src/Controller/SignalementController.php

```diff
@@ -552,7 +561,7 @@ public function suiviSignalement(
         Request $request,
         UserManager $userManager,
         SignalementDesordresProcessor $signalementDesordresProcessor,
-    ): RedirectResponse|Response {
+    ): Response {
```

Comment:
Pourquoi plus de `RedirectResponse` alors qu'il reste un `return $this->redirectToRoute('home')` ?